platforms/Windows/readme.md

```diff
@@ -165,10 +167,11 @@ msbuild %SourceRoot%\swift-installer-scripts\platforms\Windows\bundle\installer.
   -p:Configuration=Release ^
   -p:BaseOutputPath=%PackageRoot%\online\ ^
   -p:ImageRoot=%ImageRoot%\Program Files\Swift ^
+  -p:PythonRoot=%ImageRoot%\Program Files\Python-3.10.1 ^
```

Comment:
I think that we want to sink this further 1 level as `%ImageRoot%\Program Files\Swift\Python-3.10.1`. When we do that, we can remove the parameter as we only need the python _version_ as we know the image root is `%ImageRoot%\Program Files\Swift` so we can compute the path relative to that as `$(ImageRoot)\Python-$(PythonVersion)`.

Reply:
Fixed, thanks 👍